File: SchoolApiService/ViewModels/ExamScheduleStandardVM.cs
namespace SchoolApiService.ViewModels
{
    public class ExamScheduleStandardVM
    {
        public int ExamScheduleStandardId { get; set; }
        public int? ExamScheduleId { get; set; }
        public int? StandardId { get; set; }
        public string? StandardName { get; set; }
        public string? ExamScheduleName { get; set; }
        public IEnumerable<ExamSubjectVM>? ExamSubjects { get; set; } = [];
    }
}

File: SchoolAppClient.NG/src/app/Models/exam-schedule-standard-vm.ts
import { ExamSubjectVM } from "./exam-subject-vm";

export class ExamScheduleStandardVm {
  examScheduleStandardId!: number;

  examScheduleId!: number;
  standardId!: number;
  standardName!: string;
  examScheduleName!: string;
  examSubjects!: ExamSubjectVM[]
}
